pipenv/core.py

```diff
@@ -692,19 +699,78 @@ def batch_install(
     sequential_deps=None,
     extra_pip_args=None,
 ):
-    from .vendor.requirementslib.models.utils import (
-        strip_extras_markers_from_requirement,
-    )
-
     if sequential_deps is None:
         sequential_deps = []
-    failed = not retry
     deps_to_install = deps_list[:]
     deps_to_install.extend(sequential_deps)
     deps_to_install = [
         dep for dep in deps_to_install if not project.environment.is_satisfied(dep)
     ]
-    trusted_hosts = []
+    search_all_sources = project.settings.get("install_search_all_sources", False)
+    sources = get_source_list(
```

Comment:
Should this be documented in plette?
```
install_search_all_sources
```

Reply:
It is not a new feature or anything, but I am sure the plette documentation lacks it -- but are we really documenting `pipenv` implementation points in `plette` now?

Reply:
I don’t think we should document this only in plette, but it makes sense for me to add it to the spec there. Thanks for opening the issue there.